README.md

```diff
@@ -207,6 +207,24 @@ Type reset to start again and test this by adding the phrase "to eat there...":
 
 # Troubleshooting
 
+* Deploy using Cloud Foundry 'cf push' gives:
+
+``FAILED
+Could not find service <Watson_service> to bind to <Bluemix_application>``
+
+You can still use `cf push`, but realize that it is trying to bind to the services listed in the manifest.yml.
+
+So, there are 3 ways you can get this to work:
+
+    * Remove the services listed in the manifest.
```

Comment:
change indent to 2 (or less) to get your bullets back. This is now using a text box with asterisks for some reason.